rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -836,6 +836,60 @@ void testSaveOldFederation(
         );
     }
 
+    @Test
+    void savePendingFederation_postWasabi_shouldBeSavedInStorageWithFormatVersion() {
+
+        // Arrange
+
+        ActivationConfig.ForBlock wasabiActivation = ActivationConfigsForTest.wasabi100().forBlock(0L);
+        PendingFederation expectedPendingFederation = new PendingFederationBuilder().build();
+        StorageAccessor storageAccessor = new InMemoryStorage();
+
+        // Act
+
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+        federationStorageProvider.setPendingFederation(expectedPendingFederation);
+        federationStorageProvider.save(networkParameters, wasabiActivation);
+
+        PendingFederation actualPendingFederationInStorage = storageAccessor.getFromRepository(PENDING_FEDERATION_KEY.getKey(),
+            PendingFederation::deserialize);
+
+        int formatVersion = storageAccessor.getFromRepository(PENDING_FEDERATION_FORMAT_VERSION.getKey(), BridgeSerializationUtils::deserializeInteger);
+
+        // Assert
+
+        assertEquals(STANDARD_MULTISIG_FEDERATION.getFormatVersion(), formatVersion);
+        assertEquals(expectedPendingFederation, actualPendingFederationInStorage);
+
+    }
+
+    @Test
+    void savePendingFederation_preWasabi_shouldBeSavedInStorageSerializedFromBtcKeysOnly() {
+
+        // Arrange
+
+        ActivationConfig.ForBlock orchidActivation = ActivationConfigsForTest.orchid().forBlock(0L);
+        PendingFederation expectedPendingFederation = new PendingFederationBuilder().build();
+        StorageAccessor storageAccessor = new InMemoryStorage();
+
+        // Act
```

Comment:
In the Act part, should be the actual result of the method we are testing. We could avoid adding the arrange part in this section. E.g.:

// Act
`federationStorageProvider.save(networkParameters, orchidActivation);`

Reply:
Updated.